### PR TITLE
Remove non SRD monsters

### LINKED ISSUE #112: Non-SRD monsters should be removed
A few monsters are present in the database but should be removed.

So far I could only spot the Ancient Blue Dracolich, but I suspect more are present. I'll check this deeper in the following days.

### PR DESCRIPTION
**Pull request by ogregoire**
Fixes #112 
Removes the following monsters:
* Adult Blue Dracolich
* Carrion Crawler
* Cave Bear

### CODE REVIEW

**Comment by bagelbits** [comment]
You're going to hate me for this, but should we adjust the index of the rest of the monsters?

**Comment by bagelbits** [comment]
I'd highly recommend doing this programmatically if possible. Otherwise, it's going to be a pain.

**Comment by ogregoire** [comment]
@bagelbits yes it's feasible but I don't know how I could do it so that I spend less time on it than manually modifying each value. JavaScript would be a good candidate, but I'm not fluent with it. Also the URL must be adapted. And actually, this is why having unrelated numbers as keys is a bad idea. Every time we're gonna modify the number of elements in an array, we're gonna have to keep the numbers up to date. That's really not easy. But yes, I can do it. I'll try when I have time this week.

**Comment by bagelbits** [comment]
@ogregoire If you want, I can do this as a separate PR. I can probably brew up a quick script to clean it up. Feel free to cut an issue and assign it to me.

**Review by bagelbits — APPROVED** [review]
LGTM

**Comment by ogregoire** [comment]
I updated the file with the new indices. As I said, it wasn't easy. But I managed to to it.

**Comment by bagelbits** [comment]
I'm really sorry for making you do that. I'll try to figure out a better way to do that in the future. If I ever ask that again, make me write a script for you.

**Comment by ogregoire** [comment]
> I'm really sorry for making you do that. I'll try to figure out a better way to do that in the future. If I ever ask that again, make me write a script for you.

You shouldn't be sorry. I need to up my game with regards to automated changes. Currently, all the changes I do on this repo are done using regex searches and replacements.

Being a Java programmer, Json is hard to parse and several libraries are just doing too much, not giving access to basic Json functionality. I had to copy a library source code, and adapt it to not mess up the ordering of fields. But now it's done and it's in my tool box.
